stac_fastapi/pgstac/utils.py

```diff
@@ -116,7 +116,7 @@ def dict_deep_update(merge_to: Dict[str, Any], merge_from: Dict[str, Any]) -> No
             merge_to[k] = v
 
 
-def format_datetime_range(dt_range: DateTimeType) -> Union[str, Any]:
+def format_datetime_range(dt_range: Union[DateTimeType, Any]) -> Union[str, Any]:
```

Comment:
for me it seems the output should always be of type `str`

Reply:
the `input` should be `Union[DateTimeType, str]`

Reply:
should input be Union[DateTimeType, datetime]? It processes a DateTimeType or a datetime object.

Reply:
DateTimeType can be a single datetime

Reply:
No sorry it filters a string too